jena-parent/pom.xml

```diff
@@ -258,37 +257,92 @@
         <version>${ver.lucene}</version>
       </dependency>
 
-      <!-- Solr client -->
-      <!-- Exclusion of slf4j: Necessary so as to pick the version we want. 
-           solrj->zookeeper has a dependency on slf4j -->
+      <!-- For jena-spatial -->
+      <dependency>
+        <groupId>org.apache.lucene</groupId>
+        <artifactId>lucene-spatial</artifactId>
+        <version>${ver.lucene}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.apache.lucene</groupId>
+        <artifactId>lucene-spatial-extras</artifactId>
+        <version>${ver.lucene}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.locationtech.spatial4j</groupId>
+        <artifactId>spatial4j</artifactId>
+        <version>${ver.spatial4j}</version>
+      </dependency>
 
+      <!-- ES dependencies-->
       <dependency>
-        <artifactId>solr-solrj</artifactId>
-        <groupId>org.apache.solr</groupId>
-        <version>${ver.solr}</version>
+        <groupId>org.elasticsearch</groupId>
+        <artifactId>elasticsearch</artifactId>
+        <version>${ver.elasticsearch}</version>
         <exclusions>
           <exclusion>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
+            <groupId>commons-logging</groupId>
```

Comment:
Jena replaces commons-logging with org.slf4j::jcl-over-slf4j, the SLF4F adapter.  As a consequence, we have to exclude it for dependencies that pull it in.  This isn't the only case - org.apache.httpcomponents::httpclient and others.